Add info about installing pre-commit manager

AlwxSin opened the pull request:
For first time it is unclear how to use pre-commit hooks with `.pre-commit-config.yml` file.
I added info about `pre-commit` manager package to `development-process` page, where pre-commit instructions are.
But I think, that info about manager should be earlier, like at the `dependencies` page. But on that page there is info only about `python` deps.
What do you think?

sobolevn commented:
Thanks! Can you please add this line: "It comes bundled in our requirements."
So it will be clear, that you don't have to install some additional packages.

AlwxSin commented:
@sobolevn I added requested line and removed link to `pre-commit` package.
When I opened template I was 100% sure that I need to install package manually.
So there is no need to send user to `pre-commit` page if we can specify that it already installed.

sobolevn commented:
Great, thanks!
